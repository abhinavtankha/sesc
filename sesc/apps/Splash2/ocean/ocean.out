
Ocean simulation with W-cycle multigrid solver
    Processors                         : 1
    Grid size                          : 130 x 130
    Grid resolution (meters)           : 20000.00
    Time between relaxations (seconds) : 28800
    Error tolerance                    : 1e-07


                       PROCESS STATISTICS
                  Total          Multigrid         Multigrid
 Proc             Time             Time            Fraction
    0                 0                  0              -nan

                       TIMING INFORMATION
Start time                        :       1581917667
Initialization finish time        :       1581917667
Overall finish time               :       1581917667
Total time with initialization    :                0
Total time without initialization :                0
    (excludes first timestep)

