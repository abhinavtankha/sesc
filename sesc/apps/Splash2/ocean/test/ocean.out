
Ocean simulation with W-cycle multigrid solver
    Processors                         : 1
    Grid size                          : 130 x 130
    Grid resolution (meters)           : 20000.00
    Time between relaxations (seconds) : 28800
    Error tolerance                    : 1e-07


                       PROCESS STATISTICS
                  Total          Multigrid         Multigrid
 Proc             Time             Time            Fraction
    0                 0                  0              -nan

                       TIMING INFORMATION
Start time                        :       1581387195
Initialization finish time        :       1581387195
Overall finish time               :       1581387195
Total time with initialization    :                0
Total time without initialization :                0
    (excludes first timestep)

